### ISSUE TITLE
[Enhancement]: Add aria-label to wc_help_tip

### ISSUE THREAD

**Issue by nathanss** [woothemes/woocommerce#35860]
### Describe the solution you'd like

Currently, all tooltips generated with [wc_help_tip](https://github.com/woocommerce/woocommerce/blob/2a56407ba125ab281f901817af2485438c18a9b0/plugins/woocommerce/includes/wc-core-functions.php#L1601) are not visible by screen readers. 

We could make them visible to screen readers by setting tabIndex to 0 and the aria-label attribute to the same value as the tooltip text.

**Comment by barryhughes** [comment]
Thanks, that makes a lot of sense :+1:

Adding to our backlog (and if anyone is interested in submitting a PR, we'd be receptive to that).

**Comment by Sidsector9** [comment]
Hello @chihsuan I'd like to pick this up.
